components/dash/UsersTable.tsx

```diff
@@ -18,20 +18,37 @@ import {
 } from "@/components/ui/drawer";
 import UserEditForm from "@/components/dash/UserEditForm";
 import { User as UserSchema } from "@/server/schema/user";
-import { User } from "@prisma/client";
+
+import { Role, User } from "@prisma/client";
+import {UserMakeAdminButton} from "@/components/dash/UserMakeAdminButton";
+import { toast } from "../ui/use-toast";
+import { DataTableRightsSelector } from "./dataTable/DataTableRightsSelector";
+import { RotateCw } from "lucide-react";
 import { RouterOutputs } from "@/trpc/shared";
 
+
+function displayTimeFromDate(date: Date) {
```

Comment:
const {
    data: users,
    refetch,
    **dataUpdatedAt,**
  } = api.user.....useQuery(); **dataUpdatedAt,** removes the need for a function and useState rerender